package.json

```diff
@@ -1,5 +0,0 @@
-{
-  "devDependencies": {
-    "nodemon": "^3.1.7"
-  }
-}
```

Comment:
@Saurabhchaudhary9799 why did you delete this?

Reply:
@AlfiyaSiddique , I installed some node_modules in TastyTrials directory by mistake , so i deleted this. This package.json file  is not necessary.